flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNSessionFIFOITCase.java

```diff
@@ -61,6 +61,9 @@
 class YARNSessionFIFOITCase extends YarnTestBase {
     private static final Logger log = LoggerFactory.getLogger(YARNSessionFIFOITCase.class);
 
+    protected static final String VIEW_ACLS = "user groupUser";
+    protected static final String MODIFY_ACLS = "admin groupAdmin";
```

Comment:
Comma separated ?

Reply:
[yarn_config_configuration.html](https://github.com/apache/flink/pull/22509/files#diff-4cc0177d60d77c8054982fe8c4b0edcd8e521d6a6afadd48bea95486cd236ae7) says usernames should be comma separated + <Space> + group names comma separated.
So, here `user` is showing username and `groupUser` is for Group name.

Reply:
`groupUser` threw me off - `group` would be better :)

Reply:
sure, changed.